test/TemplateRenderLiquidTest.js

```diff
@@ -957,15 +1001,17 @@ test("Issue 600: Liquid Shortcode argument with underscores", async (t) => {
   );
 });
 
-test.skip("Issue 611: Run a function", async (t) => {
-  // This works in Nunjucks
+test("Issue 611: Run a function", async (t) => {
+  // function calls in Nunjucks can be replaced by custom Drops
   let tr = getNewTemplateRender("liquid", "./test/stubs/");
-
+  class CustomDrop extends Drop {
```

Comment:
is this useful at all? Liquid use a concept `Drop` to implement scenarios like this, even has a ruby like `liquidMethodMissing()` method available.

Reply:
Yeah, I like this!